### ISSUE TITLE
'readOnly' isn't a known property of 'md2-select'

### ISSUE THREAD

**Issue by yuzhva** [Promact/md2#68]
Looks like that property is missing )=

https://github.com/Promact/md2/blob/master/src/lib/select/select.ts

```
<md2-select id="uniqId" formControlName="selectName" [readonly]="!isAbleToEditForm ? true : null">
    <md2-option value="uniqValue">uniqValue</md2-option>
</md2-select>
```

**Comment by dharmeshpipariya-zz** [comment]
@YUzhva you can use 'disabled' instead of 'readonly'

**Comment by yuzhva** [comment]
@dharmeshpipariya 

Yeah - I did it. The other side of 'disabled' property is when you are trying to use it with FormBuilder.


> It looks like you're using the disabled attribute with a reactive form directive. If you set disabled to true
>       when you set up this control in your component class, the disabled attribute will actually be set in the DOM for
>       you. We recommend using this approach to avoid 'changed after checked' errors.
>        
>       Example: 
>       form = new FormGroup({
>         first: new FormControl({value: 'Nancy', disabled: true}, Validators.required),
>         last: new FormControl('Drew', Validators.required)
>       });

So I found how to use md2 date picker, just opened that issue to mentioned that this property is missed.

**Comment by dharmeshpipariya-zz** [comment]
we have fixed FormBuilder issue with #114
